app/uk/gov/hmrc/agentmapping/controller/MappingController.scala

```diff
@@ -59,51 +57,43 @@ class MappingController @Inject()(
       }
   }
 
-  def createMapping(utr: Utr, arn: Arn, unused: String): Action[AnyContent] = Action.async { implicit request =>
+  def createMapping(arn: Arn): Action[AnyContent] = Action.async { implicit request =>
     implicit val hc = fromHeadersAndSession(request.headers, None)
     authorised(AuthProviders(GovernmentGateway) and AffinityGroup.Agent)
       .retrieve(Retrievals.credentials and Retrievals.agentCode and Retrievals.allEnrolments) {
         case credentials ~ agentCodeOpt ~ allEnrolments =>
```

Comment:
lets have an auth action for this, which will provide the required account details for the controller method.